src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -431,6 +432,21 @@ export const ReserveConfiguration: React.FC<{ reserve: ComputedReserveData }> =
                   </ParentSize>
                 </ChartContainer>
               )}
+              <Box
+                sx={{ display: 'inline-flex', alignItems: 'center', pt: '42px', pb: '12px' }}
```

Comment:
For some of these `px` values, we can use the built in MUI values, which map to 1=4px.  So `pb: '12px'` can become `pb: 3` and `'42px'` can become `10.5`.

Reply:
If this page is ever re-designed, it makes sense to refactor the styling and change these. For now I think it's fine to leave it though